app/views/maintenance_tasks/runs/_run.html.erb

```diff
@@ -8,4 +8,7 @@
   <% else %>
     <%= render 'maintenance_tasks/runs/info', run: run, with_status: true %>
   <% end %>
+  <% if defined?(task) && task.csv_task? %>
+    <%= link_to('Download CSV', csv_file_task_run_path(task, run)) %>
```

Comment:
It seems like there's a way to directly link to an attachment without writing a controller action: https://guides.rubyonrails.org/active_storage_overview.html#linking-to-files
Is this something that didn't work in our case?

Reply:
Yeah, great question - I mostly just wanted more flexibility around naming the file 😄  But perhaps that's an embellishment we don't need, we can just have it use the original file name...